docs/_data/prague-2021-config.yaml

```diff
@@ -148,15 +140,6 @@ about:
   projector_ratio: "16:9"
   job_fair_room: "Sponsor area"
 
-cfp:
```

Comment:
Would deleting these cause us problems later when we rebase against the latest Portland content? Juuuuuuust checking :D

Reply:
No, we can just add them at the time.